### PR TITLE
Contact me

### PR DESCRIPTION
**Pull request by alizaamir123**
In this PR: 
Add contact us form in index.html file
and give styling in style.css file

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi Team👋👋, 

INVALID


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter checks failing🥲. Your stylint check is currently failing, kindly fix the issue and resubmit your project👍.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -165,6 +165,30 @@ <h4 class="connect">LET'S CONNECT</h4>
...
 </Section>
 
-  
+<div class="container1">
+  <h2 class="contact-us-heading">Contact me</h2>
+  <p>If you have an application you are interested
+    in developing, a feature that you need built or a project that needs coding.
+    I’d love to help with it!</p>
+<div class="form">
+  <form action="#" method="post">
```
Comment (Qoosim) [inline index.html:174]: - [ ] It is required to integrate your for with formspree service where the `action` atttibute in your form would point to. You must create an account on the [formspree](https://formspree.io/html/) site where you would be given an id that will be used in your form. the action attribute on your form will look like this :point_down: 

```
  <form action="https://formspree.io/f/{form_id}" method="post">
```
Kindly work on that.

File: index.html [hunk 2]
```diff
@@ -165,6 +165,30 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+    </label>
+    <label for="email">
+      <input type="email" id="email" name="email" placeholder="Enter your email" required>
+    </label>
+    <label for="message">
+      <textarea id="message" name="message" cols="30" rows="5" maxlength="500" placeholder="Enter your message here..." required></textarea>
+    </label>
+    <button type="submit" class="contact-btn">Get in touch</button>
+  </form>
+</div>
```
Comment (Qoosim) [inline index.html:186]: - [ ] It is required to stick to the design as close as possible. This simply means that, you're expected to produce a close replica of the design. At this moment, your design does not look like how it is on the Figma design. Kindly look into the contact me section on the Figma design and work your design alongside that design.

### Your Design
your-design

### Figma Design
temp-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @alizaamir123, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Tufahel — CHANGES_REQUESTED** [review]
Hi @alizaamir123, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted the wrong pull request.

According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) you need to submit a pull request with the implementation of the portfolio desktop version.

- [ ] Kindly open a new pull request with the implementation of the desktop version of the portfolio and ask review again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**